CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next
 - Windows: fix build error from over-specified SDK version [#3246](https://github.com/react-native-video/react-native-video/pull/3246)
 - Windows: fix `onError` not being raised [#3247](https://github.com/react-native-video/react-native-video/pull/3247)
+- Android: update isCodecSupported to return enum [#3254](https://github.com/react-native-video/react-native-video/pull/3254)
```

Comment:
would be better to clearly indicate this is a breaking change ! Thank you

Reply:
Done ✅